advocacy_docs/edb-postgres-ai/console/getstarted/index.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: EDB Postgres AI Console - Get started
+navTitle: Get started
+description: Get started with the EDB Postgres AI Console.
+deepToC: true
+---
+
+The EDB Postgres® AI Console is a web-based user interface that provides a single pane of glass for managing and monitoring EDB Postgres AI Database Cloud Service and EDB Postgres AI databases. The EDB Postgres AI Console provides a unified view of the EDB Postgres AI Database Cloud Service and EDB Postgres AI databases, allowing users to manage and monitor their databases, users, and resources from a single interface.
+
+## Accessing the EDB Postgres AI Console
+
+To access the Console, you need to have an account with the EDB Postgres AI Database Cloud Service. If you don't have an account, you can sign up for a free trial at [https://www.enterprisedb.com/edb-postgres-ai](https://www.enterprisedb.com/edb-postgres-ai).
```

Comment:
I'm checking whether this is the case and if it's how we plan to proceed even with existing customers.

Reply:
Updated the register URL and that still has the $300 offer, but just going to say sign up for now.

Reply:
Yeah, the outcome of my checking was that we're not pushing people to the $300 trial necessarily, just to login and go to the console as they will automatically have a personal organisation where they can add instances with the agent etc without entering a credit card